src/view/com/profile/ProfileSubpageHeader.tsx

```diff
@@ -72,54 +70,54 @@ export function ProfileSubpageHeader({
   }, [openLightbox, avatar])
 
   return (
-    <CenteredView style={pal.view}>
+    <CenteredView style={{ backgroundColor: tokens.color.l0 }}>
       {isMobile && (
-        <View
-          style={[
-            {
-              flexDirection: 'row',
-              alignItems: 'center',
-              borderBottomWidth: 1,
-              paddingTop: isNative ? 0 : 8,
-              paddingBottom: 8,
-              paddingHorizontal: isMobile ? 12 : 14,
-            },
-            pal.border,
-          ]}>
+        <Box
+          row aic
+          pt={web('s')}
+          pb='s'
+          px='m'
+          borderColor='l3'
+          borderBottomWidth={1}
+          gtMobile={{
+            px: 14
+          }}>
           <Pressable
             testID="headerDrawerBtn"
             onPress={canGoBack ? onPressBack : onPressMenu}
             hitSlop={BACK_HITSLOP}
-            style={canGoBack ? styles.backBtn : styles.backBtnWide}
+            w={canGoBack ? 20 : 40}
+            h={30}
+            px={canGoBack ? 0 : 6}
             accessibilityRole="button"
             accessibilityLabel={canGoBack ? 'Back' : 'Menu'}
             accessibilityHint="">
             {canGoBack ? (
               <FontAwesomeIcon
                 size={18}
                 icon="angle-left"
-                style={[styles.backIcon, pal.text]}
+                style={{ marginTop: tokens.space.s, color: tokens.color.l5 }}
```

Comment:
A wrapper around icons would be a early todo so we can just style them directly using attrs